### DISCUSSION TITLE
How to add Conditional Formatting Type dataBar

### DISCUSSION THREAD

**Discussion by BjoernBoba** [exceljs/exceljs#1704] [Q&A]
Hi @ all,

I am currently trying to add Conditional Formatting of type "dataBar" to some of my cells.
Unfortunately I am not able to make it work. Whenever I open my generated excel file I am facing an error that the document theme was removed from the workbook.

Was anyone of you able to make this work? Maybe also something similar like "ColorScale" would help as a starting point.

Thanks!

**Comment by Siemienik** [comment]
Hi @BjoernBoba, could you please add the code that you are using?

**Reply by BjoernBoba** [reply]
Hi @Siemienik, thanks for the quick reply.

My current code is
`oWorksheet.addConditionalFormatting({
				ref: 'M3:M22',
				rules: [
					{
						type: 'dataBar',
						cfvo: [{type: 'min', value: 0}, {type: 'max', value: 100}],
						style: {fill: {type: 'pattern', pattern: 'solid', bgColor: {argb: 'FF00FF00'}}}
					}
				]
			});`
			
But to be honest I guess my main issue is that I do not completely know what the cfvo parameter expects and how it behaves. Also the style part is copied from an example I found.
So for me it would be also very helpful to get some input on how the cfvo and style should look like.

Thanks a lot!
